cloud-haven-react-app/src/tokenizer.js

```diff
@@ -0,0 +1,69 @@
+// https://stackoverflow.com/questions/5010288/how-to-make-a-function-wait-until-a-callback-has-been-called-using-node-js
```

Comment:
I think we should probably take this out

Reply:
removed